mkosi

```diff
@@ -3145,17 +3151,63 @@ def run_shell(args):
 
         sys.exit(255)
 
+def run_qemu(args):
+
+    if platform.machine() == "x86_64":
+        arch_binary = "qemu-system-x86_64"
+    else:
+        arch_binary = None
+
+    if arch_binary is not None:
+        found = shutil.which(arch_binary)
+    else:
+        found = None
+
+    if found is None:
+        found = shutil.which("qemu")
+
+    if found is None:
+        found = shutil.which("qemu-kvm")
+        if found is None:
+            die("Couldn't find QEMU/KVM binary")
```

Comment:
I think it would be nice to have a command `--qemu-binary` option to be able to override the command.

Reply:
Some later PR.